### PR TITLE
Added function and fixed bug in PerfectCube.java

### PR DESCRIPTION
**Pull request by TaranjeetSinghKalsi**
Added another function to PerfectCube.java and fixed a testing mistake in line number 9

- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by siriak — CHANGES_REQUESTED** [review]
The code looks good, could you please add some JUnit tests? (see https://github.com/TheAlgorithms/Java/tree/master/src/test/java/com/thealgorithms)

**Review by TaranjeetSinghKalsi — COMMENTED** [review]
Added Junit Tests. Kindly review it.

File: src/main/java/com/thealgorithms/maths/PerfectCube.java [hunk 1]
```diff
@@ -6,12 +6,19 @@
...
         assert isPerfectCube(8);
         assert isPerfectCube(27);
+        
+        assert isPerfectCubeMathCbrt(-1);
+        assert isPerfectCubeMathCbrt(0);
+        assert isPerfectCubeMathCbrt(1);
+        assert !isPerfectCubeMathCbrt(4);
+        assert isPerfectCubeMathCbrt(8);
+        assert isPerfectCubeMathCbrt(27);
     }
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/maths/PerfectCube.java:22]: remove the main function

**Review by debasishbsws — CHANGES_REQUESTED** [review]

File: src/main/java/com/thealgorithms/maths/PerfectCube.java (hunk 1 above)
Comment (TaranjeetSinghKalsi) [reply src/main/java/com/thealgorithms/maths/PerfectCube.java]: done

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Comment by siriak** [comment]
@debasishbsws please review

**Review by debasishbsws — APPROVED** [review]
Looks good

**Pull request merged**